Sources/Utils/Backtrace.cpp

```diff
@@ -16,14 +16,16 @@
 #include <dlfcn.h>
 #include <execinfo.h>
 #elif defined(OS_WIN32)
+#include <dbghelp.h>
+#include <mutex>
+#include <sstream>
```

Comment:
Nit: would be nice to keep the C++ after the C headers

Reply:
Done.